src/lazabs/horn/preprocessor/BooleanClauseSplitter.scala

```diff
@@ -37,7 +37,7 @@ import lazabs.horn.parser.HornReader
 
```

Comment:
here you seem to have an old version of the source file?

Reply:
Yes. Now they are synchronized to master